task09-reactjs/src/components/cards/CardList.js

```diff
@@ -0,0 +1,22 @@
+import CardItem from './CardItem';
+import classes from './CardList.module.css'
+
+function CardList(props) {
+    return(
+        <div className= {classes.elementsContainer}>
+            {props.cards.map(card => (
+                <CardItem 

```

Comment:
Since most props you are passing have the same name you could do something like :
`<CardItem key= {card.name} {...card} />`
this is called prop spreading